templates/telco/index.html

```diff
@@ -0,0 +1,242 @@
+{% extends '_layout.html' %}
+
+
+{% block title %}Telco. The 5G mobile subscriber experience.{% endblock %}
+{% block meta_description %}Delighting network customers with 5G apps.
+{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1DEG0SELYXIJCdqwaJOWM6R1yCiTnEub4kaK0aX4bT2I/edit#heading=h.44dtjnxyfix2#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<div class="wrapper">
+    <div id="main-content" class="inner-wrapper">
+        <section class="p-strip--suru-half-and-half-reversed is-dark">
+            <div class="row u-equal-height u-vertically-center">
+              <div class="col-6">
+                <h1>
+                    The 5G mobile <br>subscriber experience
+                </h1>
+                <p class="p-heading--four">
+                    Delighting network customers with 5G apps
+                </p>
+              </div>
+              <div class="col-6 u-align--center u-hide--small">
+                    {{ image (
+                        url="https://assets.ubuntu.com/v1/55cff1c7-Anbox_Telco_5G.svg",
+                        alt="",
+                        width="152",
+                        height="195",
+                        hi_def=True,
+                        loading="auto"
+                        ) | safe
+                    }}
+              </div>
+            </div>
+        </section>
+
+        <section class="p-strip--light">
+            <div class="row u-equal-height u-vertically-center">
+                <div class="col-6">
+                    <p>
+                        5G lifts broadband and latency limitations in mobile networks. </br>
+                        However, building scalable infrastructure to deliver 5G apps to mobile subscribers is expensive. Anbox Cloud is a scalable platform to host 5G apps in telco RANs in the cloud and at the edge.
+                    </p>
+                    <p>
+                        Leading telco service providers are rolling out 5G networks to offer a new generation of broadband access to their subscribers. The virtually unlimited bandwidth and latency of 5G unlock new possibilities. Telcos that create innovative value-added services will be winners in the 5G race.
+                    </p>
+                    <p>
+                        With 5G, elastic computing in the cloud becomes available to mobile users at zero latency. However, mobile handset capabilities are restricted and will remain a bottleneck. Offloading compute and storage  to the cloud via 5G networks could be a way to circumvent these limitations.
+                    </p>
+                    <p>
+                        <a href="">
+                            Learn about value-added telco services with Anbox Cloud >
```

Comment:
I've asked Sam in the copy doc who we can ask for the correct link for this

```suggestion
                            Learn about value-added telco services with Anbox Cloud&nbsp;&rsaquo;
```

Reply:
I'm still waiting on the link, can we omit it for now and come back to it?

Reply:
Yep, leave a comment on the copy doc to remind the author of the content.